index.html

```diff
@@ -1,12 +1,336 @@
+                />
+              </svg>
+            </a>
+          </li>
+
+          <li>
+            <a href="#">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="14"
+                height="14"
+                fill="currentColor"
+                class="bi bi-facebook"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
+                />
+              </svg>
+            </a>
+          </li>
+          <li><span>Logout</span></li>
+        </ul>
+      </div>
+    </header>
+
+    <div class="container" id="hamburger">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+    </div>
+
+    <nav id="nav" class="hide">
+      <div class="logo">
+        <a href="index.html">
+          <img src="images/logo.png" class="logo-img" alt="logo" />
+        </a>
+        <a class="navbar-brand" href="index.html">Foodland Restaurant</a>
+      </div>
+      <ul>
+        <li class="px-4">
+          <a aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="px-4">
+          <a aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="px-4">
+          <a href="#">Speakers</a>
+        </li>
+        <li class="px-4">
+          <a href="#">Partners</a>
+        </li>
+        <li class="px-4">
+          <a href="#">Contact</a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- Navbar End -->
+    <!-- body section -->
+    <section class="intro">
+      <div class="row">
+        <div class="col-lg-12 col-md-12 col-sm-12">
+          <div class="mx-5 mt-5">
+            <div class="main-intro">
+              <h2 class="intro-message">"You are what you eat!"</h2>
+              <h1>Welcome to Foodland Restaurant</h1>
+            </div>
+            <div class="card">
+              <div class="card-body">
+                <p class="card-text">
+                  Foodland Restaurant is a Nigerian Restaurant located in
+                  Sapele, Delta State. They serve craving delicious Nigerian
+                  traditional food, English dishes and so on.. At Foodland
+                  Restaurant your satisfaction is our number one priority. We
+                  live to serve you.
+                </p>
+              </div>
+            </div>
+            <h2 class="intro-date">2015.10.15(THU)~16(FRI)</h2>
+            <span>@Okpe road, Beside the Oando Petrol Station in Sapele</span>
+          </div>
+        </div>
+        <div class="intro"></div>
+      </div>
+    </section>
```

Comment:
Kindly add some spacing in order to match the required design

Your Design
![image](https://user-images.githubusercontent.com/66112760/170843707-4ef19431-3b72-45bc-8c84-d26237a4b4f2.png)

Figma Design
![image](https://user-images.githubusercontent.com/66112760/170843721-9687be1b-ed86-47c0-b0e4-0604d7e177c7.png)